lib/modules/uscore_v3.1.1/profile_definitions/bp_definitions.rb

```diff
@@ -53,6 +87,32 @@ class USCore311BpSequenceDefinitions
           }.freeze,
           {
             path: 'dataAbsentReason'
```

Comment:
Since we exclude Obervation.value[x], should we also exclude Observation.databasentReason?